README.md

```diff
@@ -90,6 +94,71 @@ They currently do not support (in order of importance):
     )
     ```
 
+* To depend on external libraries, you have two options: vendoring or external
```

Comment:
This is a pretty long section for a README, but the information is useful.
I don't know what other projects do, but can a 'Vendoring.md' be created with this info and linked to?